dev-libs/libgnt/libgnt-2.14.1.recipe

```diff
@@ -0,0 +1,87 @@
+SUMMARY="Text-mode UI toolkit"
+DESCRIPTION="libgnt is a GLib-based ncurses toolkit for creating text-mode \
+graphical user interfaces quickly and easily."
+HOMEPAGE="https://pidgin.im/"
+COPYRIGHT="2006-2021 libgnt contributors"
+LICENSE="GNU GPL v2"
+REVISION=1
+SOURCE_URI="http://downloads.sourceforge.net/project/pidgin/libgnt/$portVersion/libgnt-$portVersion.tar.xz"
+CHECKSUM_SHA256="5ec3e68e18f956e9998d79088b299fa3bca689bcc95c86001bc5da17c1eb4bd8"
+SOURCE_DIR="libgnt-$portVersion"
+PATCHES="libgnt-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+versionCompat="$portVersion compat >= ${portVersion%%.*}"
```

Comment:
Try to use libVersion and libVersionCompat as described in the guidelines: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#using-the-libversioncompat-user-variable